anda/system/submarine/submarine.spec

```
@@ -3,6 +3,10 @@
 %elifarch aarch64
 %global arch arm64
 %endif
+# do not strip binaries
+%global __os_install_post %{nil}
```

I'm unsure where you got this from, this should work:
```suggestion
%define __strip /bin/true
```